### PR TITLE
AO3-4773 Extend tests for known_issues_controller

### PR DESCRIPTION
**Pull request by pixelred**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4773

## Purpose

add steps for deleting known issues in admin_steps.rb
add test for deleting known issues in admin_post_issues.feature
add a few lines just to cover "show" button as well

### CODE REVIEW

**Comment by pixelred** [comment]
i dunno if it'll help the coverage at all? but i can add another line for when you click the "show" button

File: features/step_definitions/admin_steps.rb [hunk 1]
```diff
@@ -261,14 +261,21 @@ def update_settings(settings)
 
 When /^I edit known issues$/ do
   step(%{I am logged in as an admin})
-    step( %{I follow "Admin Posts"})
+    step(%{I follow "Admin Posts"})
```
Comment (zz9pzza) [inline features/step_definitions/admin_steps.rb]: Can we fix the indent here ?

File: features/step_definitions/admin_steps.rb [hunk 2]
```diff
@@ -261,14 +261,21 @@ def update_settings(settings)
...
     step(%{I follow "Known Issues" within "#main"})
     step(%{I follow "Edit"})
     step(%{I fill in "known_issue_title" with "More known problems"})
     step(%{I fill in "content" with "This is a bit of a problem, and this is too"})
     step(%{I press "Post"})
 end
 
+When /^I delete known issues$/ do
+  step(%{I am logged in as an admin})
+    step(%{I follow "Admin Posts"})
```
Comment (zz9pzza) [inline features/step_definitions/admin_steps.rb]: Can we fix the indent here ?

**Comment by pixelred** [comment]
of course! i think that was all of them :v:

File: features/step_definitions/admin_steps.rb (hunk 2 above)
Comment (sarken) [inline features/step_definitions/admin_steps.rb]: Also, jsyk, the `()` around the step are unnecessary -- just `step %{I follow "Admin Posts"}` is good.

Comment (pixelred) [inline features/step_definitions/admin_steps.rb]: okay! i'll do that now

for some reason travis is giving the `expected to find text "Actually, I meant something different"` failure again... that's the second time today

File: features/step_definitions/admin_steps.rb [hunk 3]
```diff
@@ -260,13 +260,20 @@ def update_settings(settings)
...
+  step %{I am logged in as an admin}
+  step %{I follow "Admin Posts"}
+  step %{I follow "Known Issues" within "#main"}
+  step %{I follow "Edit"}
+  step %{I fill in "known_issue_title" with "More known problems"}
+  step %{I fill in "content" with "This is a bit of a problem, and this is too"}
+  step %{I press "Post"}
+end
+
+When /^I delete known issues$/ do
```
Comment (pixelred) [inline features/step_definitions/admin_steps.rb]: what D:

Comment (sarken) [inline features/step_definitions/admin_steps.rb]: I think you can ignore Hound on this one.

**Comment by pixelred** [comment]
passed 👍

**Pull request merged**